lib/framework/action.dart

```diff
@@ -482,7 +482,7 @@ abstract class EnsembleAction {
     if (action is YamlMap) {
       ActionType? actionType = ActionType.values.from(action.keys.first);
       dynamic payload = action[action.keys.first];
-      if (actionType != null && payload is YamlMap) {
+      if (actionType != null && payload is YamlMap?) {
```

Comment:
what's the question mark for?

Reply:
GPT taught me the ?. Learning new stuff every day.

Reply:
So it also matches null, which was the regression